lib/skate_web/controllers/notification_read_states_controller.ex

```diff
@@ -5,7 +5,10 @@ defmodule SkateWeb.NotificationReadStatesController do
   alias SkateWeb.AuthManager
 
   def update(conn, %{"new_state" => new_read_state, "notification_ids" => notification_ids}) do
-    username = AuthManager.Plug.current_resource(conn)
+    username =
+      conn
+      |> AuthManager.Plug.current_resource()
+      |> AuthManager.username_from_resource()
```

Comment:
this duplicated chain is temporary: once all tokens in circulation use the new format, we can access the username using `%{username: username} = AuthManager.Plug.current_resource(conn)`